web/sharings/sharings_test.go

```diff
@@ -117,8 +117,8 @@ func TestSharings(t *testing.T) {
 	tsB.Config.Handler.(*echo.Echo).HTTPErrorHandler = errors.ErrorHandler
 
 	// Prepare another instance for the replicator tests
```

Comment:
I think we no longer need this instance in this file (only used in `replicator_test.go`)